components/crud-web-apps/volumes/backend/entrypoint.py

```diff
@@ -7,29 +7,16 @@
 log = logging.getLogger(__name__)
 
 
-def get_config(mode):
-    """Return a config based on the selected mode."""
-    config_classes = {
-        config.BackendMode.DEVELOPMENT.value: config.DevConfig,
-        config.BackendMode.DEVELOPMENT_FULL.value: config.DevConfig,
-        config.BackendMode.PRODUCTION.value: config.ProdConfig,
-        config.BackendMode.PRODUCTION_FULL.value: config.ProdConfig,
-    }
-    cfg_class = config_classes.get(mode)
-    if not cfg_class:
-        raise RuntimeError("Backend mode '%s' is not implemented. Choose one"
-                           " of %s" % (mode, list(config_classes.keys())))
-    return cfg_class()
-
-
 APP_NAME = os.environ.get("APP_NAME", "Volumes Web App")
 BACKEND_MODE = os.environ.get("BACKEND_MODE",
                               config.BackendMode.PRODUCTION.value)
 UI_FLAVOR = os.environ.get("UI_FLAVOR", "default")
 PREFIX = os.environ.get("APP_PREFIX", "/")
+METRICS = bool(os.environ.get("METRICS", False))
```

Comment:
Do we need to set a default value both in each specific app and in the config? Maybe we should do it like `backend_mode`
```
BACKEND_MODE = os.environ.get("BACKEND_MODE",
                              config.BackendMode.PRODUCTION.value)
```

Reply:
Actually I moved the `self.METRICS = bool(os.environ.get("METRICS", True))` to config __init__, since the logic is for all apps sames (check if env is set to 1, true, ...).